### ISSUE TITLE
mos 2.19.1 (prebuilt windows version) process not closed

### ISSUE THREAD

**Issue by Sekilsgs2** [mongoose-os/mos#56]
2.19.1 version running in browser. (i'm using chrome)
When i'm close mos tab - the process continues to run in the task list.
And when i'm run mos again - it won't start - need manual kill old mos process.
OS - windows 10 x64.

**Comment by Harvie** [comment]
This is obviously not a bug. Just put the adress to your bookmarks so you can reopen it without restarting the mos tool.
Web browsing simply does not work like that. Imagine that google would shut down their servers because you've closed tab.

**Comment by Sekilsgs2** [comment]
> This is obviously not a bug. Just put the adress to your bookmarks so you can reopen it without restarting the mos tool.
> Web browsing simply does not work like that. Imagine that google would shut down their servers because you've closed tab.
> Also you can't run it twice, because only one process can bind to port. If you need to run two mos tools (for two different mos projects), then you need to use two different port numbers. If you use single project, then just reopen that address.

If i'm close chrome - two process stay in running! One - its main mos and second - its serial terminal process. I'm using ft232 usb uart and this mos tool not working normal - main process cant access serial if second process active - need close second process and run commands in mos tool. 
And second process never closing if i'm close mos tool.
I'm thing this is because developers make tests/fixes only for esp devices and scored for everything else.

**Comment by Harvie** [comment]
> If i'm close chrome - two process stay in running! 

Once again. This is not a bug.

**Comment by Sekilsgs2** [comment]
> > If i'm close chrome - two process stay in running!
> 
> Once again. This is not a bug. Expected behaviour.

This is a bug - this is incorrect behavior. If I need to restart the utility, should I go into the process manager and manually kill? I do not understand what you want to prove.

**Comment by rojer** [comment]
i agree that it's a bug. closing tab should close the background process, at least by default. having a flag to make it persist would be nice but default behavior should be for it to exit.

**Comment by rakeshyannamani** [comment]
![image](https://user-images.githubusercontent.com/6128116/123147981-53641300-d47d-11eb-8841-5033a0cdc163.png)

Is there a solution for this error ?
